types/react-mailchimp-subscribe/index.d.ts

```diff
@@ -17,35 +17,37 @@ export interface NameFormFields extends EmailFormFields {
 }
 
 export interface ClassicFormFields extends NameFormFields {
-    "BIRTHDAY[month]": number;
-    "BIRTHDAY[day]": number;
+    'BIRTHDAY[month]': number;
+    'BIRTHDAY[day]': number;
 }
 
 // library default form just sends EMAIL
 export type DefaultFormFields = EmailFormFields;
 
-export interface ResponseArgs {
```

Comment:
Perhaps consider adding `export type ResponseArgs = ErrorResponseArgs | SuccessResponseArgs` to ease this transition for consumers. That would reduce the chances that this becomes a breaking change for someone.

Reply:
Tks for the suggestion! Done.